### PR TITLE
Support new critical level severity

### PR DESCRIPTION
**Pull request by yahavi**
- [x] All [tests](https://github.com/jfrog/jfrog-vscode-extension#building-and-testing-the-sources) passed. If this feature is not already covered by the tests, I added new tests.
- [x] I used `npm run format` for formatting the code before submitting the pull request.
-----

Change icons to support new critical level severity:
* Current low severity icon changed to medium severity.
* Current medium severity icon changed to high severity.
* Current high severity change to critical.
* Add new low severity icon.

To summarize, after this PR, the icons will look as the following:
image

The hover icons:
image

### CODE REVIEW

**Comment by Or-Geva** [comment]
The filter by issue severity is broken? e.g. 
image

**Review by Or-Geva — APPROVED** [review]
